nan.h

```diff
@@ -173,7 +173,7 @@ NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
 }
 
 /* io.js 1.0  */
-#if NODE_MODULE_VERSION >= IOJS_1_0_MODULE_VERSION \
+#if NODE_MODULE_VERSION >= NODE_0_12_MODULE_VERSION \
  || NODE_VERSION_AT_LEAST(0, 11, 15)
```

Comment:
Now I know why it does not work. nw.js claims to be node 0.11.13, but it is not! On top of that it bundles a version of v8 different from that in node 0.11.13.